tests/adversarial_namespace/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+include(CMakeParseArguments)
```

Comment:
Thanks for including comprehensive tests!
I would have liked it even better though if it had simply all been put in one test to curb the explosion of files/schemas/binaries.

Reply:
I think the primary problem with putting them all in one test is the custom pointer/string as an attribute vs as a flag (the code is handled slightly differently), but it should be possible to combine it into two tests rather than eight.